kdc/kerberos5.c

```diff
@@ -931,6 +931,7 @@ struct kdc_patypes {
 #define PA_SYNTHETIC_OK	4
     krb5_error_code (*validate)(astgs_request_t, const PA_DATA *pa);
 };
+#define PA_NOT_IN_FAST	8 /* don't announce in fast */
```

Comment:
Also could you please move this to be after `PA_SYNTHETIC_OK`.

Reply:
Done.